specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-10-20/examples/MyWorkbookUpdate.json

```diff
@@ -23,7 +23,7 @@
     }
   },
   "responses": {
-    "200": {
+    "201": {
```

Comment:
There are quite many breaking changes, as you may noticed. Per rule made by breaking change review board, you will need follow the process defined in https://aka.ms/AzBreakingChangesPolicy to request breaking approval. Thus, for this issue, I would suggest you to change service implementation instead. That will also resolve the S360 issue, and there's no breaking change.

Reply:
So it sounds like I understood the S360 error correctly. 

there are many breaking changes because the previous swagger definition was defined incorrectly to start with. that is why i'm requesting a breaking change exemption so that we can start regularly maintaining it and keeping it up to date. 

It seemed like the full breaking change approval process would take a long time and since the s360 deadline was approaching, I thought that the exception was the right request since these changes are not actually breaking changes - they are correcting the incorrect existing swagger definition. But if there's no way around it and we have to go through the  whole process, then that is fine. I'm just concerned that we'd just be out of SLA for the long duration of the approval process.